Source/Core/AudioCommon/DPL2Decoder.cpp

```diff
@@ -154,20 +155,20 @@ static float* DesignFIR(unsigned int *n, float* fc, float opt)
 	if (o)
 	{
 		w[end] = fc1 * w[end] * 2.0f;
-		g=w[end];
+		g = w[end];
 	}
 
 	// Create filter
 	for (u32 i = 0; i < end; i++)
 	{
-		t1 = (float)(i+1) - k2;
-		w[end-i-1] = w[*n-end+i] = float(w[end-i-1] * sin(k1 * t1)/(M_PI * t1)); // Sinc
-		g += 2*w[end-i-1]; // Total gain in filter
+		t1 = (float)(i + 1) - k2;
+		w[end - i - 1] = w[*n - end + i] = float(w[end - i - 1] * sin(k1 * t1)/(M_PI * t1)); // Sinc
+		g += 2*w[end - i - 1]; // Total gain in filter
 	}
 
 
 	// Normalize gain
-	g=1/g;
+	g = 1/g;
```

Comment:
I'm split on whether to put a space between division and multiplication operators. When dealing with numbers and single letter variables, sometimes it looks better when stuck together. I'm just not sure...